### DISCUSSION TITLE
نحوه نصب یک نسخه دلخواه مرزبان و همچنین آپدیت مرزبان

### DISCUSSION THREAD

**Discussion by farshadl** [Gozargah/Marzban#900] [General]
من ۲ تا سوال دارم. ممنون میشم راهنمایی کنید.
۱) من میخوام نسخه قدیمی ۰.۴.۷ مرزبان را روی سرورم نصب کنم. نمیخوام آخرین نسخه را نصب کنم. از چه دستوری باید استفاده کنم؟
۲) اگه من یک نسخه قدیمی پنل مرزبان روی سرورم نصب باشه و بخوام اون را آپدیت کنم، از چه دستوری باید استفاده کنم؟

**Comment by waxilu** [comment]
answer of 2nd question: `marzban update`

**Comment by Aghil1991** [comment]
خیلی ساده، داکر رو در مسیر:

/opt/marzban/docker-compose.yml

باز کنید و به جای latest یا dev ورژن مورد نظر خودتون رو به شکل زیر قرار بدین.

image: gozargah/marzban:v0.5.2

بعد سیو کنید و مرزبان رو ریستارت کنید.

marzban restart
